### PR TITLE
Updated Python license for 2021.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Review by jacobtylerwalls — APPROVED** [review]

File: LICENSE.python [hunk 1]
```diff
@@ -263,3 +274,17 @@ FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
 WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
 ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT
 OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+ZERO-CLAUSE BSD LICENSE FOR CODE IN THE PYTHON DOCUMENTATION
+----------------------------------------------------------------------
```
Comment (pope1ni) [inline LICENSE.python:279]: Fix length of this heading underline?

**Review by pope1ni — APPROVED** [review]
Thanks.

File: LICENSE.python (hunk 1 above)
Comment (felixxm) [reply LICENSE.python:279]: I don't think we should change anything. It's a copy of Python license https://github.com/python/cpython/blob/main/LICENSE.

Comment (pope1ni) [reply LICENSE.python:279]: I figured. 👍🏻

**Comment by felixxm** [comment]
Thanks both for checking :+1:

**Pull request merged**